config/application.rb

```diff
@@ -41,5 +42,10 @@ class Application < Rails::Application
 
     # Path within public/ where assets are compiled to
     config.assets.prefix = '/government-frontend'
+
+    # Do not swallow errors in after_commit/after_rollback callbacks.
+    # config.active_record.raise_in_transactional_callbacks = true
+
+    GdsApi.config.always_raise_for_not_found = true
```

Comment:
I think this should belong in `initializers/gds_api.rb` since it's related to configuration of the library and not the application.